README.adoc

```diff
@@ -47,7 +47,14 @@ If you want to generate the Dockerfiles for a specific components of the XebiaLa
 
 [source,shell,subs="verbatim,attributes"]
 ----
-$ {executable} render --xl-version <version> --product {xl-deploy,xl-release}
+$ {executable} render --xl-version <version> --product xl-deploy --product xl-release
+----
+
+If you want to specify different base image organisation than default one "xebialabs" in which generated docker images will pull from you can use `--pull-registry`, This may be needed while generating Docker images for XL Jetpack Release and xl JetPack Deploy from xebialabsunsupported organisation.
+
+[source,shell,subs="verbatim,attributes"]
+----
+$ {executable} render --xl-version <version> --product xl-jetpack-deploy --pull-registry xebialabsunsupported
```

Comment:
If we are sure that everytime it will be the same repo, i mean one you pull from and push too .. which might be the case now then yes .. but my idea is to make it more flexible for future use too.

For example .. if you want to pull from xebialabsunsupported, build image and push to local registry xl-docker.xebialabs.com or the otherway around.
Also for other users like if they wants to pull from xebialabs registry and push to their local one.

Also my suggestion is to have 2 flags here --pull-registry (which i have this pull request opened for) and --push-registry (which can be just rename of --registry)

Reply:
Currently the flexibility is there, as they're 2 commands, `render` and `build` ;)

Reply:
Aha okay got your point so you mean to use --registry flag, with render it will mean registry you want to pull from and with build it will mean registry you want to push to .. right ?

Reply:
Yes

Reply:
Okay will do this .. Thanks 👍 :-)